Finicky/Finicky/AppDelegate.swift

```diff
@@ -192,13 +192,17 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSUserNotificationCenterDele
         }
 
         for browser in browsers {
-            let apps = NSRunningApplication.runningApplications(withBundleIdentifier: browser.bundleId)
-            if !apps.isEmpty {
-                let app: NSRunningApplication = apps[0]
-                let bundleIdentifier = app.bundleIdentifier
-                if bundleIdentifier != nil {
-                    return browser
+            if let bundleId = browser.bundleId {
+                let apps = NSRunningApplication.runningApplications(withBundleIdentifier: bundleId)
+                if !apps.isEmpty {
+                    let app: NSRunningApplication = apps[0]
+                    let bundleIdentifier = app.bundleIdentifier
+                    if bundleIdentifier != nil {
+                        return browser
+                    }
                 }
+            } else if browser.appPath != nil {
```

Comment:
Oh yeah, I thought I left a review comment about this but I guess not. I could not find any easy way to tell if an app is running by path. So that left me with two options:

if there are multiple matching options, one specified by bundle id and one by path, then:

1) always open one specified by path
2) never open the one specified by path

I chose #1 because the "specify by path" is an override in my mind. However, that may not be what you intended. What do you think it should do?

Also, I am not an advanced macOS app developer by any means. Do you know of a way to tell if an application is running given its path? Maybe some way to hook into activity monitor API? in the info of an app, the `Open Files and Ports` tab contains this info:

```
cwd
/
txt
/Applications/Google Chrome.app/Contents/MacOS/Google Chrome
```

Which, if I could read it, would help me determine if the app is running.

Reply:
I changed it to option 2 in my PR as that seems more in line with your intention.